### PR TITLE
Start of New Documentation

### LINKED ISSUE #47: bdlib has released patches
Per the last comment here:

https://github.com/bdew-minecraft/bdlib/issues/57

bdlib has patched the issue in two of the older releases, so this information can be added to the README.

### PR DESCRIPTION
**Pull request by Aiidoneus**
This is the start of the new documentation while helping people understand what mods have been fixed by us and mod developers. By any means is it perfect but I think it does the job well. This closes #47

### CODE REVIEW

**Comment by Aiidoneus** [comment]
I know bdew is working on adding fixes for most versions of his mod. I will wait and add them all at once once 1.12 is fixed an pushed out to the public See #47

File: README.md [hunk 1]
```diff
@@ -38,49 +38,16 @@ We developed a patcher that attempts to fix all currently known affected mods (l
...
-- [Thaumic Tinkerer](https://www.curseforge.com/minecraft/mc-mods/thaumic-tinkerer) (Fixed in version 2.3-138 for Minecraft 1.7.2, versions for 1.6-1.6.4 remain affected, [relevant commit](https://github.com/Thaumic-Tinkerer/ThaumicTinkerer/commit))
-- [Tough Expansion](https://www.curseforge.com/minecraft/mc-mods/tough-expansion)
-- [ttCore](https://www.curseforge.com/minecraft/mc-mods/ttcore) (Only affects versions for Minecraft 1.7.10)
+Unlike the blog post stated above, there are plenty of mods that are and could be affected by this issue. Although some of them are already fixed in their latest versions, these mods were exploitable in at least one older verison. It is to be expected that several modpacks over the years just are not maintained anymore, but are still popular and loved within the community. Keeping this in-mind, we are trying to help those people who still love running those modpacks and strive to keeping them safe as they play.
```
Comment (Exaxxion) [inline README.md:41]: I believe the last sentence of this paragraph should instead say "striving to keep" to maintain consistency with "trying to help" earlier in the sentence.

File: README.md [hunk 2]
```diff
@@ -38,49 +38,16 @@ We developed a patcher that attempts to fix all currently known affected mods (l
...
+Unlike the blog post stated above, there are plenty of mods that are and could be affected by this issue. Although some of them are already fixed in their latest versions, these mods were exploitable in at least one older verison. It is to be expected that several modpacks over the years just are not maintained anymore, but are still popular and loved within the community. Keeping this in-mind, we are trying to help those people who still love running those modpacks and strive to keeping them safe as they play. 
+
+**KEEP IN MIND THAT THIS LIST IS DEFINITELY NOT COMPLETE! THESE ARE JUST THE MODS WE ARE CURRENTLY AWARE OF!** At least Curseforge is already investigating the issue internally so we can maybe get a nearly complete list of vulnerable mods and versions in the future.
+
+We have moved our affected mods list to another location! See the link below:
+
+**[Affected Mods List](docs/mods.md)**
+
+This list will change frequently as we find more mods that could have vulnerabilities and developers adding patches to their own mods. If you want to help us in keeping this list up-to-date, please feel free to contribute to it!
```
Comment (Exaxxion) [inline README.md:49]: "developers adding patches" -> "developers add patches" to keep consistent with "as we find".

Probably don't need "to it" after "contribute" as well but that's a stylistic choice.

**Review by Exaxxion — COMMENTED** [review]
Assuming chart is accurate, that part LGTM. Please see review comments for suggestions re: the ReadMe.

**Comment by miozune** [comment]
AsieLib and TecTech from https://github.com/dogboy21/serializationisbad/pull/33 are not listed

**Comment by Aiidoneus** [comment]
> AsieLib and TecTech from #33 are not listed

I re-add those soon.

File: README.md (hunk 1 above)
Comment (Aiidoneus) [reply README.md]: Fixed via https://github.com/dogboy21/serializationisbad/pull/48/commits/fbaa9497d1e30fc83f632b7238b7678c56a56a20

File: README.md (hunk 2 above)
Comment (Aiidoneus) [reply README.md]: Fixed via https://github.com/dogboy21/serializationisbad/pull/48/commits/fbaa9497d1e30fc83f632b7238b7678c56a56a20

**Comment by Aiidoneus** [comment]
> AsieLib and TecTech from #33 are not listed

Fixed via https://github.com/dogboy21/serializationisbad/pull/48/commits/fbaa9497d1e30fc83f632b7238b7678c56a56a20

**Comment by miozune** [comment]
`TechTech` -> `TecTech`
Also [AsieLib](https://github.com/dogboy21/serializationisbad/blob/master/serializationisbad.json#L152-L160) and [TecTech](https://github.com/dogboy21/serializationisbad/blob/master/serializationisbad.json#L97-L105) are already patched

**Comment by Aiidoneus** [comment]
Thanks for the catch. This is what I get for doing this at 3 in the morning. 
Fixed via https://github.com/dogboy21/serializationisbad/pull/48/commits/c8eb2014ddbd0023ce0258fb3011462d8c5bb630

Updated README in various places

File: docs/mods.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+|                                                    [mxTune](https://www.curseforge.com/minecraft/mc-mods/mxtune)                                                     |                                                                                                             Only affects Minecraft versions<br>`1.12` to `1.16.5 `                                                                                                             |                                                  Nonthing is fixed in the affected versions                                                   |                ✅ <br> [Project Patch](https://github.com/dogboy21/serializationisbad/blob/master/serializationisbad.json#L263-L271)                 |                                                                                          |                                                                                                                    |
```
Comment (dogboy21) [inline docs/mods.md:28]:
```suggestion
|                                                    [mxTune](https://www.curseforge.com/minecraft/mc-mods/mxtune)                                                     |                                                                                                             Only affects Minecraft versions<br>`1.12` to `1.16.5`                                                                                                             |                                                  Nothing is fixed in the affected versions                                                   |                ✅ <br> [Project Patch](https://github.com/dogboy21/serializationisbad/blob/master/serializationisbad.json#L263-L271)                 |                                                                                          |                                                                                                                    |
```

**Review by dogboy21 — CHANGES_REQUESTED** [review]

**Review by dogboy21 — APPROVED** [review]

**Pull request merged**